### ISSUE TITLE
Deactivate site on subdomain / other domain

### ISSUE THREAD

**Issue by JoryHogeveen** [polylang/polylang#10]
Deactivating a website requires a user to log in. But when you use a subdomain or other domain and you use the default domain to access the admin this is a bit of a hassle.

Ofcourse it is logical that this won't work since you don't have the login cookies when on a other or subdomain but maybe there is something to do about this?

Perhaps a temprary metakey with the users IP address that removes itself after logout?

**Comment by Chouby** [comment]
Could you please clarify what you mean by deactivating a website?

**Comment by JoryHogeveen** [comment]
Sorry, I meant deactivating a language.

**Comment by Chouby** [comment]
So, you mean that you have issues to de-activate a language when using multiple domains. Normally you should be able to de-activate any language (except the default one) from the Languages panel of the default site. Note that you need to log out to see the effect, as an admin ( or any user having the edit-posts capability ) can still visit the site as if the language wasn't deactivated.

**Comment by JoryHogeveen** [comment]
Correct, the "problem" is as follows.

- I have the admin on the main language (ofcourse).
- When I add a language on a different domain all goes fine.
- But when I disable that language I can't view the translated website/domain (even though I am logged in on the main language).

It's clear to me that this bug is hard to fix since it required accessing logged-in cookies from an other domain because otherwist WP (and PLL) won't know it you are logged in.

In short:
- I am logged in on www.maindomain.com
- I have a translation on www.translation.nl
- When I disable the translation while I'm logged in on www.maindomain.com I can't view www.translation.nl

**Comment by Chouby** [comment]
Yes. I understand now and you are right. This indeed should be fixed when I will provide login synchronization (I am currently working on it).

**Comment by JoryHogeveen** [comment]
Ok! If you need any help (dev or testing) let me know!
